user/languages/python.md

```diff
@@ -241,3 +248,30 @@ For a real world example, see [getsentry/sentry](https://github.com/getsentry/se
 - [dstufft/slumber](https://github.com/dstufft/slumber/blob/master/.travis.yml)
 - [dreid/cotools](https://github.com/dreid/cotools/blob/master/.travis.yml)
 - [twisted/klein](https://github.com/twisted/klein/blob/master/.travis.yml)
+
+{% if site.data.language-details.python-versions.size > 0 %}
+## Python versions
+
+These archives are available for on-demand installation.
+
+{: #python-versions-table}
+| Release | Arch | Name |
+| :------------- | :------------- | :------- |{% for file in site.data.language-details.python-versions %}
+| {{ file.release }} | {{ file.arch }} | {{ file.name }} |{% endfor %}
```

Comment:
"Archives" are implementation detail and would only sidetrack users. Better specify just version IDs as they should go into `python:`.